Loop/View Controllers/PredictionTableViewController.swift

```diff
@@ -94,7 +94,7 @@ class PredictionTableViewController: LoopChartsTableViewController, Identifiable
         }
     }
 
-    let glucoseChart = PredictedGlucoseChart()
+    let glucoseChart = PredictedGlucoseChart(predictedGlucoseBounds: nil)
```

Comment:
Not clamped here?

Reply:
Correct. The request for this view is that it transparently displays the algorithm and its results.